TAF/testScenarios/functionalTest/V2-API/core-data/event/POST-negative.robot

```diff
@@ -19,13 +19,15 @@ ErrEventPOST001 - Create events fails (Event ID Conflict)
     And Should Return Content-Type "application/json"
     And Item Index 0,1,2 Should Contain Status Code "201" And id
     And Item Index 3 Should Contain Status Code "409" And no id
+    And Response Time Should Be Less Than "1200"ms
 
 ErrEventPOST002 - Create events fails (Bad Events)
     ${bad_property}=  Create List  no_event  no_deviceName  no_origin  no_readings  no_id  bad_id
     FOR  ${property}  IN  @{bad_property}
          Given Generate Bad Event With ${property}
          And Create Events
          Then Should Return Status Code "400"
+         And Response Time Should Be Less Than "1200"ms
```

Comment:
could this "1200"ms be something like ${default_response_time_threshold} and define 1200 in other place?

Reply:
Added the default threshold variable